src/utility/Assert.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2019 Ole Kliemann, Malte Kliemann
```

Comment:
```suggestion
/* Copyright 2020 Ole Kliemann, Malte Kliemann
```

Not sure about this, but it's fine according to [How do you update copyright notices?](https://softwareengineering.stackexchange.com/questions/34189/how-do-you-update-copyright-notices)